src/components/SearchResults.jsx

```diff
@@ -93,118 +80,205 @@ const SearchResults = () => {
-																	return (
-																			<Button disabled
-																							variant="contained"
-																			>
-																				Pending
-																			</Button>
-																	);
-																default:
-																	return '-';
-															}
-														})()}</TableCell>
-														{/*<TableCell>{formatDate(dataItem.update_date)}</TableCell>*/}
-													</TableRow>
-												);
-											})
-										}
-									</TableBody>
-								</Table>
-							</TableContainer>
-							{data.length > rowsPerPage && (
-									<TablePagination className={classes.tablePagination}
-																	 rowsPerPageOptions={[10, 20, 50, {label: 'All', value: -1}]}
-																	 component="div"
-																	 count={data.length}
-																	 rowsPerPage={rowsPerPage}
-																	 page={page}
-																	 onChangePage={handleChangePage}
-																	 onChangeRowsPerPage={handleChangeRowsPerPage}
-									/>
-							)}
-						</>
-					:
-					""
+																dataItem.doi
+														:
+														'-'
+													}
+												</TableCell>
+												<TableCell className={classes.columnLidvid}>{dataItem.lidvid}</TableCell>
+												<TableCell>{dataItem.title}</TableCell>
+												<TableCell className={classes.columnStatus}>{massageStatus(dataItem.status.toLowerCase())}</TableCell>
+												<TableCell>{(() => {
+													switch (dataItem.status.toLowerCase()) {
+														case 'draft':
+														case 'reserved':
+															return (
+																	<Button color="primary"
+																		variant="contained"
+																		onClick={(event) => handleReleaseClick(dataItem.lidvid)}
+																	>
+																		Release
+																	</Button>
+															);
+														case 'registered':
+															return (
+																	<Button color="primary"
+																		variant="contained"
+																		onClick={(event) => handleReleaseClick(dataItem.lidvid)}
+																	>
+																		Update
+																	</Button>
+															);
+														case 'review':
+															return (
+																	<Button disabled
+																		variant="contained"
+																	>
+																		Pending
+																	</Button>
+															);
+														default:
+															return '-';
+													}
+												})()}</TableCell>
+											</TableRow>
+										);
+									})
+								}
+							</TableBody>
+						</Table>
+					</TableContainer>
+					{data.length > rowsPerPage && (
+						<TablePagination className={classes.tablePagination}
+							rowsPerPageOptions={[10, 20, 50, {label: 'All', value: -1}]}
+							component="div"
+							count={data.length}
+							rowsPerPage={rowsPerPage}
+							page={page}
+							onChangePage={handleChangePage}
+							onChangeRowsPerPage={handleChangeRowsPerPage}
+						/>
+					)*/}
+				</div>
```

Comment:
Since the majority of this is commented out, this whole chunk could be `null` or its contents deleted.